webpage/video_tutorials.md

```diff
@@ -21,6 +21,10 @@ layout: single
 - [How to use the GLOS data portal](https://www.youtube.com/playlist?list=PLEK-mxRrHSZ5LkSSUyYHEy4lEXnEdNJJc)
 - [How to use the GLOS boater's tool](https://www.youtube.com/playlist?list=PLEK-mxRrHSZ6rH1qCpfpD6-rSljE8fbUC)
 
+![OOI logo]({{ site.url }}/{{ site.baseurl }}/assets/OOI_Logo.png) OOI Demonstration Videos
```

Comment:
@jbosch-noaa I created a new category to add OOI videos, does that make sense?

Reply:
Sure.  That's a good idea.  I'm sure there will be more OOI videos. 😃